app/src/modules/insights/routes/dashboard.vue

```diff
@@ -82,9 +82,12 @@
 			<insights-navigation />
 		</template>
 
+		<span v-if="errors">{errors}</span>
+
 		<v-workspace
+			:loading="loading"
```

Comment:
Instead of passing `loading` here, I think we should `v-if` it on the component render down on L100